File: problemsets/Codeforces/Go/A595.go
/**
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 */
package main

import "fmt"

func main() {
	var n, m int
	r := 0
	fmt.Scan(&n, &m)
	for i:=0;i<n*m;i++ {
		var a, b int
		fmt.Scan(&a,&b)
		if a+b>0 {
			r++
		}
	}
	fmt.Print(r)
}

File: problemsets/Codeforces/Python/A595.py
"""
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 """
import re
n,m=map(int,input().split());r=0
for _ in [0]*n:
  r += sum(1 for x in re.split('(\d \d)',input()) if x.count('1')) 
print(r)